README.md

```diff
@@ -1,191 +1,58 @@
-</p>
+---
 
-Ably Chat is a set of purpose-built APIs for a host of chat features enabling you to create 1:1, 1:Many, Many:1 and Many:Many chat rooms for
-any scale. It is designed to meet a wide range of chat use cases, such as livestreams, in-game communication, customer support, or social
-interactions in SaaS products. Built on [Ably's](https://ably.com/) core service, it abstracts complex details to enable efficient chat
-architectures.
+# Ably Chat Swift SDK
 
-Get started using the [📚 documentation](https://ably.com/docs/products/chat).
+Ably Chat is a set of purpose-built APIs for a host of chat features enabling you to create 1:1, 1:Many, Many:1 and Many:Many chat rooms for any scale. It is designed to meet a wide range of chat use cases, such as livestreams, in-game communication, customer support, or social interactions in SaaS products. Built on [Ably's](https://ably.com/) core service, it abstracts complex details to enable efficient chat architectures.
 
-![Ably Chat Header](/images/ably-chat-github-header.png)
+---
 
-## Supported Platforms
-
-- macOS 11 and above
-- iOS 14 and above
-- tvOS 14 and above
-
-## Requirements
-
-Xcode 16.1 or later.
-
-## Installation
-
-The SDK is distributed as a Swift package and can hence be installed using Xcode (search for `github.com/ably/ably-chat-swift` package) or by adding it as a dependency in your package’s `Package.swift`:
-
-```swift
-.package(url: "https://github.com/ably/ably-chat-swift", from: "0.4.0")
-```
-
-## Supported chat features
-
-This project is under development so we will be incrementally adding new features. At this stage, you'll find APIs for the following chat
-features:
-
-- Chat rooms for 1:1, 1:many, many:1 and many:many participation.
-- Sending and receiving chat messages.
-- Online status aka presence of chat participants.
-- Chat room occupancy, i.e total number of connections and presence members.
-- Typing indicators
-- Room-level reactions (ephemeral at this stage)
-
-If there are other features you'd like us to prioritize, please [let us know](https://forms.gle/mBw9M53NYuCBLFpMA).
-
-## Usage
-
-You will need the following prerequisites:
-
-- An Ably account
-
-  - You can [sign up](https://ably.com/signup) to the generous free tier.
+## Getting started
 
-- An Ably API key
-  - Use the default or create a new API key in an app within
-    your [Ably account dashboard](https://ably.com/dashboard).
-  - Make sure your API key has the
-    following [capabilities](https://ably.com/docs/auth/capabilities): `publish`, `subscribe`, `presence`, `history` and
-    `channel-metadata`.
+Everything you need to get started with Ably:
 
-To instantiate the Chat SDK, create an [Ably client](https://ably.com/docs/getting-started/setup) and pass it into the Chat constructor:
+* [About Ably Chat.](https://ably.com/docs/chat)
+* [Getting started with Ably Chat in Swift.](https://ably.com/docs/chat/getting-started/swift)
```

Comment:
This guide doesn't seem to exist; I assume we don't want to merge until it does?

Reply:
Yes. This is in prodcution @umair-ably

Reply:
[SDK setup for Swift ](https://ably.com/docs/chat/setup?lang=swift)added instead